src/PromptUsageStrategy.ts

```diff
@@ -0,0 +1,53 @@
+import { CopilotSettings } from "@/settings/SettingsPage";
```

Comment:
For consistency can we name this file `promptUsageStrategy.ts` without the capitalization.